payments/src/main/java/it/gov/pagopa/payments/model/spontaneous/DebtorModel.java

```diff
@@ -40,6 +41,7 @@ public class DebtorModel implements Serializable {
 
     private String region;
 
+    @Pattern(regexp="[A-Z]{2}", message="The country must be reported with two capital letters (example: IT)")
```

Comment:
remove `payments` folder according to https://github.com/pagopa/pagopa-gpd-payments too